.travis.yml

```diff
@@ -3,16 +3,17 @@ dist: trusty
 language: ruby
 rvm: 2.4.1
 env: CODECLIMATE_REPO_TOKEN=611cda7a6df35c4ca4df08319797b6eba033ca5f61068f90072cb148f71714c8
-addons:
-  apt:
-    sources: google-chrome
-    packages: google-chrome-stable
-before_script:
+install:
+  - whereis firefox | tr ' ' '\n' | grep bin | xargs sudo rm
   - sudo apt-get update
-  - sudo apt-get install chromium-chromedriver
-  - export PATH=$PATH:/usr/lib/chromium-browser
+  - sudo apt-get install firefox
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.16.1/geckodriver-v0.16.1-linux64.tar.gz
+  - tar -zxvf geckodriver*
+  - sudo mv geckodriver /usr/local/bin
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - bundle install
```

Comment:
I've never needed to include this in a Rails `.travis.yml` before, but it just suddenly stopped happening. Not sure why.